.github/workflows/test-python-package.yml

```diff
@@ -8,6 +8,7 @@ on:
     branches:
       - 'main'
       - 'feature/**'
+      - 'dev'
```

Comment:
hmm, not sure why this would be here if dev already has this, are you sure you rebased on to dev

Reply:
+1